eks/cluster.tf

```diff
@@ -138,15 +138,46 @@ resource "aws_iam_policy" "vm-service-ec2" {
 EOF
 }
 
+resource "aws_iam_policy" "external_dns" {
+  name        = "${var.basename}-external-dns-policy"
+  description = "IAM Policy to allow the external-dns service to set DNS records"
+  policy      = <<-EOF
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ChangeResourceRecordSets"
```

Comment:
Are you positive that we don't try to create a hosted zone for the external URL? I'm not sure myself right now

Reply:
I am positive. In either case the external-dns service should never be creating hosted zones. It should only be modifying records. This permission set comes from the recommendations of the external-dns docs here:

https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#iam-policy